sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -530,6 +530,10 @@ export class EventHubReceiver extends LinkEntity {
     }
   }
 
+  getSequenceNumber(): number {
+    return this._checkpoint.sequenceNumber;
```

Comment:
`getSequenceNumber` doesnt make it clear that we are getting the sequenceNumber from the checkpoint

Also, it doesnt look like we are using the `offset` and `enqueuedTime` information on the `_checkpoint` variable. If this is true, then why not make `_checkPoint` point to the sequenceNumber directly?

Then we can replace `getSequenceNumber` method with a readonly property `checkpoint`

Reply:
`this._checkpoint` was already implemented, so I just used it to get the sequenceNumber. I've made the changes now, but lets sync up if we also want to cleanup existing code in the same pull request. I'd prefer if we had done this in a separate PR for cleaning up existing code.